build.gradle.kts

```diff
@@ -84,3 +85,21 @@ publishing {
         }
     }
 }
+
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    kotlin {
+        ktlint().editorConfigOverride(
+            mapOf(
+                "indent_size" to "4",
+                "disabled_rules" to "package-name"
```

Comment:
When I run `spotlessApply` `Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.` is printed 900 times:
```sh
➜  tests-gantt-chart git:(adding_spotless) ./gradlew spotlessApply --rerun-tasks | grep "is deprecated" -m 1
Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.
➜  tests-gantt-chart git:(adding_spotless) ./gradlew spotlessApply --rerun-tasks | grep "is deprecated" -c
900
```

Reply:
Fixed, more info in comment in `build.gradle.kts`